README.md

```diff
@@ -93,7 +92,9 @@ per platform, and can also be configured for cross-compilation.
 System-wide installation of the resulting sentry library is also possible via
 CMake.
 
-Building the Crashpad Backend requires a `C++14` compatible compiler.
+Depending on the platform and backend, the prerequisites to building differ. You will always need CMake to actually build the code. Additionally, when using crashpad, `zlib` is required. On Linux and macOS `curl` is a prerequisite.
```

Comment:
This is more or less duplicate information from `CONTRIBUTING.md`, but split up a bit for different configs. @supervacuus is this how you envisioned replicating the prerequisites into the readme?